index.html

```diff
@@ -16,7 +17,7 @@
             </nav>
         </div>
     </header>
-    <section>
+    <section class="head">
```

Comment:
- [OPTIONAL] I suggest you make some changes to your `head` section to stick as much as possible to the Figma design, you can take look at your design and Figma design 😉 

Your design

<img width="221" alt="Screen Shot 2022-09-22 at 13 40 54" src="https://user-images.githubusercontent.com/74501523/191737977-a1a1dd98-a677-453f-9643-26221cbebfb3.png">

Figma design

<img width="352" alt="Screen Shot 2022-09-22 at 13 38 21" src="https://user-images.githubusercontent.com/74501523/191738017-76901206-1db6-4858-b1c3-f91215203670.png">